docs/SONATA_DEVELOPER_GUIDE.md

```diff
@@ -557,6 +568,8 @@ The namespace of parameters depends on model_type, and are defined as follows.
 
 **For point_process models**, it is the namespace of the point_process/artificial cell mechanism.
 
+**For point_neuron models**, it is the namespace of the neuron parameter dictionary.
```

Comment:
I was doing some editing and I thought that keeping this new entry and removing "For point_process models" would be enough, but one line up (which doesn't appear in the diff) there's text for point_soma models. Where does this point_soma model type come form @markovg? It hasn't been listed before.

Reply:
I'd like to unblock and merge this review, @markovg, is there anybody that I can ask for clarifications about where does `point_soma` come from and whether we can remove it.